### PR TITLE
[onert] Add an optional input to ir::ResizeNearestNeighbor

### PR DESCRIPTION
**Pull request by ragmani**
For issue #4053

This commit adds an optional input to ir::ResizeNearestNeighbor.
  - Adds an optional input
  - Enable nnapi tests for this operation

Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

File: tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu [hunk 1]
```diff
@@ -110,6 +110,78 @@ GeneratedTests.relu6_quant8_1
...
+GeneratedTests.resize_nearest_neighbor_scale_nchw_8
+GeneratedTests.resize_nearest_neighbor_scale_nchw_quant8_8
+GeneratedTests.resize_nearest_neighbor_zero_sized_nhwc
+GeneratedTests.resize_nearest_neighbor_zero_sized_nhwc_quant8
+GeneratedTests.resize_nearest_neighbor_zero_sized_nchw
+GeneratedTests.resize_nearest_neighbor_zero_sized_nchw_quant8
+GeneratedTests.resize_nearest_neighbor_zero_sized_nhwc_2
+GeneratedTests.resize_nearest_neighbor_zero_sized_nhwc_quant8_2
+GeneratedTests.resize_nearest_neighbor_zero_sized_nchw_2
+GeneratedTests.resize_nearest_neighbor_zero_sized_nchw_quant8_2
```
Comment (wateret) [inline tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:184]: Wdouldn't this mean these tests are not tested? I thought this was a skiplist(block-list).

Comment (ragmani) [reply tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:184]: I intended these tests are not tested.

**Review by hseok-oh — APPROVED** [review]
LGTM

@ragmani Please update visitor `OperationDumper` for `ResizeNearestNeighbor` later (on other PR if you want)

**Comment by ragmani** [comment]
> Please update visitor OperationDumper for ResizeNearestNeighbor later (on other PR if you want)

I just added it.

**Review by hseok-oh — APPROVED** [review]
LGTM

File: tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu (hunk 1 above)
Comment (wateret) [reply tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:184]: Then how are tests enabled? I only see disabled tests.

Comment (ragmani) [reply tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:184]: The backend cpu and acl_neon don't support this operation yet. So this file disables all tests for this operation.

Comment (wateret) [reply tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:184]: That must be it is enabled only for `armv7l-linux.acl_cl`, sorry I missed it.

**Pull request merged**